yodeploy/application.py

```diff
@@ -105,15 +105,15 @@ def hook(self, hook, target, repository, version):
         if target:
             cmd += ['--target', target]
         cmd += [self.appdir, version]
-        try:
-            subprocess.check_call(cmd, env={
-                    'PATH': os.environ['PATH'],
-            })
-        except subprocess.CalledProcessError:
+
+        cmd_proc = subprocess.Popen(
+            cmd, env={'PATH': os.environ['PATH']}, close_fds=False)
+        cmd_proc.wait()
+        tlss.shutdown()
```

Comment:
Maybe this should be in a try/except/finally block in case `Popen` raises an `OSError` but `cmd` looks like something we know for sure exists so maybe not...